src/SFA.DAS.EmployerIncentives.Functions.PaymentsProcess/Activities/CalculateDaysinLearning.cs

```diff
@@ -0,0 +1,29 @@
+﻿using Microsoft.Azure.WebJobs;
+using Microsoft.Azure.WebJobs.Extensions.DurableTask;
+using Microsoft.Extensions.Logging;
+using SFA.DAS.EmployerIncentives.Abstractions.Commands;
+using SFA.DAS.EmployerIncentives.Commands.ApprenticeshipIncentive.CalculateDaysInLearning;
+using System.Threading.Tasks;
+
+namespace SFA.DAS.EmployerIncentives.Functions.PaymentsProcess.Activities
+{
+    public class CalculateDaysInLearning
+    {
+        private readonly ICommandDispatcher _commandDispatcher;
+        private readonly ILogger<CalculateDaysInLearning> _logger;
+
+        public CalculateDaysInLearning(ICommandDispatcher commandDispatcher, ILogger<CalculateDaysInLearning> logger)
+        {
+            _commandDispatcher = commandDispatcher;
+            _logger = logger;
+        }
+
+        [FunctionName(nameof(CalculateDaysInLearning))]
+        public async Task Create([ActivityTrigger] CalculateDaysInLearningInput input)
+        {
+            _logger.LogInformation($"Calculating DaysinLearning for apprenticeship incentive id {input.ApprenticeshipIncentiveId}", new { input.ApprenticeshipIncentiveId });
```

Comment:
don't think we can use string interpolation in logger

Reply:
Just implemented as per the others.  Please advise how it should be done.

Reply:
@MichaelYoung1981 do we need a var here?

Reply:
and no 💲

Reply:
It needs to not use $ and pass the variables as params

Reply:
"pass the variables as params" ? Please provide an example.  As per pervious comment it has been done as per the other logging and therefore is currently consistent with the rest of the code.

Reply:
Done